soc/silabs/silabs_siwx91x/siwg917/nwp.c

```diff
@@ -35,8 +35,14 @@ int siwg91x_get_nwp_config(int wifi_oper_mode, sl_wifi_device_configuration_t *g
 		.boot_config = {
 			.feature_bit_map = SL_SI91X_FEAT_SECURITY_OPEN | SL_SI91X_FEAT_WPS_DISABLE,
 			.tcp_ip_feature_bit_map = SL_SI91X_TCP_IP_FEAT_EXTENSION_VALID,
-			.custom_feature_bit_map = SL_SI91X_CUSTOM_FEAT_EXTENSION_VALID,
-			.ext_custom_feature_bit_map = SL_SI91X_EXT_FEAT_XTAL_CLK,
+			.custom_feature_bit_map = SL_SI91X_CUSTOM_FEAT_EXTENSION_VALID |
+				SL_SI91X_CUSTOM_FEAT_RTC_FROM_HOST,
+			.ext_custom_feature_bit_map = SL_SI91X_EXT_FEAT_XTAL_CLK |
+				SL_SI91X_EXT_FEAT_1P8V_SUPPORT |
+				SL_SI91X_EXT_FEAT_DISABLE_XTAL_CORRECTION |
+				SL_SI91X_EXT_FEAT_NWP_QSPI_80MHZ_CLK_ENABLE |
+				SL_SI91X_EXT_FEAT_FRONT_END_SWITCH_PINS_ULP_GPIO_4_5_0 |
+				SL_SI91X_EXT_FEAT_FRONT_END_INTERNAL_SWITCH,
```

Comment:
You can align the values. Either:
```
			.custom_feature_bit_map = SL_SI91X_CUSTOM_FEAT_EXTENSION_VALID |
						  SL_SI91X_CUSTOM_FEAT_RTC_FROM_HOST,
```
or:
```
			.custom_feature_bit_map = 
				SL_SI91X_CUSTOM_FEAT_EXTENSION_VALID |
				SL_SI91X_CUSTOM_FEAT_RTC_FROM_HOST,
```

Reply:
One thing that might help some editors to auto-align this, is to put the value inside parentheses, i.e.:
```c
	.ext_custom_feature_bit_map = (SL_SI91X_EXT_FEAT_XTAL_CLK |
				       SL_SI91X_EXT_FEAT_1P8V_SUPPORT |
				       ...
				       SL_SI91X_EXT_FEAT_FRONT_END_INTERNAL_SWITCH),